kuma/users/templates/users/email/welcome/plain.ltxt

```diff
@@ -30,7 +30,7 @@
 {{ _('Want to talk to someone about MDN? There are a few ways you can do that:') }}
 
 {# L10n: This is an email. Whitespace matters! #}
-* {{ _('Mailing list: {list_link}')|f(list_link=add_utm('https://lists.mozilla.org/listinfo/dev-mdc', 'welcome')) }}
+* {{ _('Mailing list:') }} {{ add_utm('https://lists.mozilla.org/listinfo/dev-mdc', 'welcome') }}
```

Comment:
Yeah, the order of words in sentences, especially when it comes to colons is important. Some languages just don't use western style listings with a colon